src/test/resources/petriflow_test.xml

```diff
@@ -21,7 +21,7 @@
         </caseLogic>
     </roleRef>
     <roleRef>
-        <id>default</id>
+        <id>defaultRole</id>
```

Comment:
leave both new `defaultRole` and old roleRef with id `default` to test id validation